src/main/java/frc/robot/subsystems/ColorPanelSubsystem.java

```diff
@@ -0,0 +1,148 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.ColorSensorV3;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorMatch;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color;
+import frc.robot.posev2.WheelColorValue;
+import edu.wpi.first.wpilibj.DriverStation;
+
+public class ColorPanelSubsystem extends BaseSubsystem{
```

Comment:
please add code into SubsystemManager that reads your field value and updates the FieldPose

https://github.com/entech281/Robot2020/blob/8958bb6fe3471f7e9172e19ef76b05a7ca1a8250/src/main/java/frc/robot/subsystems/SubsystemManager.java#L79,